### ISSUE TITLE
SA1209 - Missing file-scoped namespace support

### ISSUE THREAD

**Issue by ylabade** [DotNetAnalyzers/StyleCopAnalyzers#3439]
SA1209 UsingAliasDirectivesMustBePlacedAfterOtherUsingDirectives is not fired when a file-scoped namespace is used.

Example:
```cs
namespace WebApi.Controllers;

using System;
using SystemIO = System.IO; // SA1209 should be raised here
using Microsoft.AspNetCore.Mvc;

[Route("[controller]")]
public class WeatherForecastController : ControllerBase
{
}
```

Environment:
- .NET 6
- C# 10
- StyleCop 1.2.0-beta.376
